go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -42,7 +40,7 @@ const (
 	// 5ms, 10ms, 20ms, 40ms, 80ms, 160ms, 320ms, 640ms, 1.3s, 2.6s, 5.1s, 10.2s, 20.4s, 41s, 82s
 	maxRetries = 15
 
-	controllerName = "ovn-lb-controller"
+	controllerName = "controlplane"
```

Comment:
@zshi-redhat I don't think we want to change this? It's used for more things than the event recorder.

Reply:
Fixed to use original name `ovn-lb-controller`.

@dcbw  do you see there is any dependency on the event source name? 
By sharing the [eventRecorder](https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/util/kube.go#L272) with master, the even source name is changed to `controlplane` for service controller.

Reply:
I think that's a tiny bit unfortunate, but OK.